### PR TITLE
Added homepage background change. Closed #3

### LINKED ISSUE #3: Add homepage background change

### PR DESCRIPTION
**Pull request by Andrey31ts**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -35,8 +35,9 @@
 			<div class="btn-group dropstart">
 				<button type="button" class="btn btn-light dropdown-toggle" data-bs-toggle="dropdown"
 						aria-expanded="false">
-					<i class="bi bi-back"></i>
+					<i class="bi bi-back" ></i>
```
Comment (TopAcademy2022) [inline src/index.html:38]: Revert.

File: src/index.html [hunk 2]
```diff
@@ -57,6 +58,37 @@
 						</a>
 					</li>
 				</ul>
+
```
Comment (TopAcademy2022) [inline src/index.html:61]: Remove spaces.

File: src/script.js [hunk 3]
```diff
@@ -381,8 +381,31 @@ let ChangeBackgroundColor = (newBackgroundColor) => {
 		navbarMenu.classList.remove(currentBackgroundColor);
 		navbarMenu.classList.add(newBackgroundColor);
 	}
+
```
Comment (TopAcademy2022) [inline src/script.js:384]: Remove space.

File: src/script.js [hunk 4]
```diff
@@ -381,8 +381,31 @@ let ChangeBackgroundColor = (newBackgroundColor) => {
...
+		navbarMenu.style.backgroundRepeat = null;
+		navbarMenu.style.backgroundSize = null;
+		navbarMenu.style.backgroundPosition = null;
+	}
+
+	if (newBackgroundColor != currentBackgroundColor) {
+		navbarMenu.classList.remove(currentBackgroundColor);
+		navbarMenu.classList.add(newBackgroundColor);
+	}
+
```
Comment (TopAcademy2022) [inline src/script.js:403]: Remove spaces.

File: src/script.js [hunk 5]
```diff
@@ -398,4 +421,21 @@ let LoadImageAsBackgroundColor = (eventOpenFile) => {
 
 	let openImageInput = document.getElementById('file-open-image');
 	openImageInput.value = null;
+}
+
```
Comment (TopAcademy2022) [inline src/script.js:425]: Remove spaces.

File: src/script.js [hunk 6]
```diff
@@ -398,4 +421,21 @@ let LoadImageAsBackgroundColor = (eventOpenFile) => {
...
+	reader.readAsDataURL(eventOpenFile.files[0]);
+
+	reader.onload = function () {
+		navbarMenu.style.backgroundImage = `url(${reader.result})`;
+		navbarMenu.style.backgroundRepeat = 'no-repeat';
+		navbarMenu.style.backgroundSize = 'cover';
+		navbarMenu.style.backgroundPosition = 'center center';
+	};
+
+	let openImageInput = document.getElementById('file-open-b');
```
Comment (TopAcademy2022) [inline src/script.js:439]: Rename Id.

File: src/style.css [hunk 7]
```diff
@@ -1,8 +1,9 @@
-#file-open, #file-open-image {
+#file-open, #file-open-image, #file-open-b {
```
Comment (TopAcademy2022) [inline src/style.css:1]: Rename id.

File: src/style.css [hunk 8]
```diff
@@ -1,8 +1,9 @@
...
 	display: none;
+
```
Comment (TopAcademy2022) [inline src/style.css:3]: Remove space.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

File: src/script.js [hunk 9]
```diff
@@ -381,6 +381,26 @@ let ChangeBackgroundColor = (newBackgroundColor) => {
 		navbarMenu.classList.remove(currentBackgroundColor);
 		navbarMenu.classList.add(newBackgroundColor);
 	}
+
```
Comment (TopAcademy2022) [inline src/script.js:384]: Remove space.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 10]
```diff
@@ -58,6 +58,24 @@
...
+					<li class="d-flex justify-content-evenly">
+						<a class="dropdown-item bg-light border border-dark" style="width: 40;height: 44;margin: 4;" onclick="ChangeBackgroundColorBody('bg-light')"></a>
+						<a class="nav-link" href="#">
+							<label for="file-open-body">
+								<i class="bi bi-arrow-up-square-fill h1"></i>
+							</label>
+							<input class="dropdown-item"  id="file-open-body" type="file" onchange="LoadImageAsBackgroundColorBody(this)">
+						</a>
+					</li>
+
```
Comment (TopAcademy2022) [inline src/index.html:76]: Remove space.

**Review by TopAcademy2022 — CHANGES_REQUESTED** [review]

**Review by Mi-stik31 — APPROVED** [review]

**Pull request closed without merging**